LICENSE

```diff
@@ -1,14 +1,29 @@
-Copyright (c) 2005-2016, Regents of the University of California, the University of North Carolina at Chapel Hill, and the Data Intensive Cyberinfrastructure Foundation
-All rights reserved.
+BSD 3-Clause License
 
-Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
+Copyright (c) 2005-2024, The University of North Carolina at Chapel Hill
```

Comment:
i don't think we can drop the other two copyright holder notices.   we *could* put them on separate lines, noting their increasing age...

Reply:
Oh really? I had no idea.

Should the year(s) be identical?

Reply:
Updated.